app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -53,6 +59,10 @@ def courses
       def sanitize(course_code)
         course_code.strip.upcase
       end
+
+      def existing_course_option_ids
+        application_form.application_choices.pluck(:course_option_id)
```

Comment:
I think what we're trying to do here is stop the same course from being added twice, rather than course_option (although that is covered too).

Maybe something like:

```
def existing_course_ids
  application_form.application_choices.map(:course_option).pluck(:course_id)
end
```

And then above on line 21:

`.reject { |course_option| existing_course_ids.include?(course_option.course_id) }`

Reply:
Yup good spot. Have made the change